dsp/templates/template_v2.py

```diff
@@ -1,45 +1,32 @@
 from collections import namedtuple
 import re
-
+from typing import Union, Any
 import dsp
+from dsp.primitives.demonstrate import Example
+from .utils import passages2text, format_answers
```

Comment:
review notes: removed redundant function definitions of passages2text and format_answers